.semaphore/semaphore-scheduled-builds.yml

```diff
@@ -702,6 +702,40 @@ blocks:
           - sudo journalctl > logs/journalctl.txt
           - artifact push job --expire-in 1d logs
 
+- name: 'OpenStack integration (Yoga)'
+  run:
+    when: "true or change_in(['/networking-calico/'])"
+  dependencies: ["Prerequisites"]
+  task:
+    agent:
+      machine:
+        type: e1-standard-4
+        os_image: ubuntu2004
+    prologue:
+      commands:
+      - cd networking-calico
+    jobs:
+      - name: 'Unit and FV tests (tox) on Yoga'
+        commands:
+          - ../.semaphore/run-and-monitor tox.log make tox-yoga
+      - name: 'Mainline ST (DevStack + Tempest) on Yoga'
+        commands:
+          - dpkg -l "python*" || true
```

Comment:
What's the purpose of this command? I can't see a `-l` flag specified in the dpkg man page.

Reply:
It means to list the installed packages matching "python*".  I am surprised it's not in the man page!

I can take this line out now.  It was debugging for a problem with the "python3-wrapt" package.  It was pre-installed on the Semaphore node with a different version from what OpenStack needs, and that was causing the DevStack setup to fail.  The fix (2 lines below) is to uninstall the existing version first.